src/index.html

```diff
@@ -2,16 +2,193 @@
+            >
+          </label>
+
+        </div>
+
+        <div class="inputBlock">
+          <label for="acceptAgree">I accept the term of the agreement
+            <input
+              id="acceptAgree"
+              type="checkbox"
+              name="acceptAgree"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldsetBlock">
+        <legend>Registration</legend>
+
+        <div class="inputBlock">
+          <label for="email">E-mail:
+            <input
+              id="email"
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="inputBlock">
+          <label for="password">Password:
+            <input
+              id="password"
+              type="password"
+              name="password"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldsetBlock">
+        <legend>An interesting fact about you</legend>
+
+        <div class="inputBlock">
+          <label for="loveCats">Do you love cats?
+            <input
+              type="radio"
+              name="loveCats"
+              value="yes"
+            >
+          </label> Yes
+
+          <input
+            type="radio"
+            name="loveCats"
+            value="no"
+          >
+        </label> No
+        </div>
+
+        <div class="inputBlock">
+          <label for="favColor">What is your favorite color? </label>
+          <input
+            id="favColor"
+            type="color"
+            name="favoriteColor"
+          >
+        </div>
+
+        <div class="inputBlock">
+          <label for="timeToBad">What time do you go to bad? </label>
+          <input
+            id="timeToBad"
+            type="time"
+            name="timeToBad"
+          >
+        </div>
+
+        <div class="inputBlock">
+          <label for="favoriteCar">What are your favorite brand of cars? </label>
+          <select
+            name="favoriteCar"
+            id="favoriteCar"
+            size="4"
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">lada</option>
+            <option></option>
+          </select>
+        </div>
+
+        <div class="inputBlock">
+          <label for="workRate">How do you rate our work? </label>
+          <input
+            id="workRate"
+            type="range"
+            name="workRate"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldsetBlock">
+        <legend>Additional info:</legend>
+        <div class="inputBlock">
+          <label for="comment">Comments:
+            <textarea
+              name="comment"
+              id="comment"
+              cols="20"
+              rows="2">
+            </textarea>
```

Comment:
you don't need blank characters between your closing and opening tags of textarea cause they will render on a page as it's content
```suggestion
            <textarea
              name="comment"
              id="comment"
              cols="20"
              rows="2"
           ></textarea>
```